### ISSUE TITLE
Disclosure Accessibility isues: missing some or all of the required children

### ISSUE THREAD

**Issue by bennymi** [CaptainCodeman/svelte-headlessui#57]
Hi, I've noticed this error pop up when running a lighthouse report on the [disclosure component example](https://captaincodeman.github.io/svelte-headlessui/example/disclosure/):

`Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children.`

![image](https://github.com/CaptainCodeman/svelte-headlessui/assets/20195536/9be285a3-9dc4-489b-acb6-1f451448abb5)

It's probably because of the `role="menu"` that is being applied by the panel, but I'm not sure which other children are required.

**Comment by CaptainCodeman** [comment]
Thanks, I'll look into it - my initial reaction is that the menu role is probably being set when it shouldn't be.

**Comment by CaptainCodeman** [comment]
Yes, that was it - thanks again for the report!
